internal/gps/version.go

```diff
@@ -274,6 +282,14 @@ func (v branchVersion) Pair(r Revision) PairedVersion {
 	}
 }
 
+func (v branchVersion) equals(c Constraint) bool {
+	v2, ok := c.(branchVersion)
+	if !ok {
+		return false
+	}
+	return v == v2
```

Comment:
The idea was to have a method for verifying the cache serialization strategy, since `==` and `reflect.DeepEquals` are too strict. I have renamed the method to `identical`, and expanded the doc.